app/Models/Forum/Topic.php

```diff
@@ -455,7 +484,7 @@ public function setFirstPostCache()
 
     public function setLastPostCache()
     {
-        $lastPost = $this->posts()->last()->first();
+        $lastPost = $this->posts()->orderBy('post_id', 'desc')->first();
```

Comment:
last time I checked the original version didn't work and just returned the first element anyways